pygments/formatters/latex.py

```diff
@@ -97,6 +97,10 @@ def escape_tex(text, commandprefix):
 
 STYLE_TEMPLATE = r'''
 \makeatletter
+%% workaround a minted <= v2.5 issue, which loads this file with - having
+%% catcode letter; also with _ having catcode 11 but this does not affect us
+\edef\%(cp)s@restore@catcode@of@minus@sign{\catcode`\noexpand\-=\the\catcode`\-\relax}
+\catcode`\-=12
```

Comment:
As minted loads this file with `\endlinechar=-1` setting it would have been better here to add explicitly `\relax` after `12`. This is cosmetic as next line starts with a `\def` which does not expand hence the TeX parsing of the digits will end correctly.

Just mentioning this out of professional sense of duty... (there is no higher level LaTeX interface to set a catcode, so I used the core level TeX syntax, but one has to be aware of some things) (I would have probably used core TeX syntax anyhow anyway as I like it)

Reply:
It's certainly interesting to read :laughing: